data/xml/2020.findings.xml

```diff
@@ -3591,6 +3591,7 @@
       <doi>10.18653/v1/2020.findings-emnlp.296</doi>
       <revision id="1" href="2020.findings-emnlp.296v1" hash="3f9e2b0b"/>
       <revision id="2" href="2020.findings-emnlp.296v2" hash="626fee76" date="2021-03-26">The acronym for the dataset introduced was revised in order to avoid confusion with the name of an existing organization.</revision>
+      <revision id="3" href="2020.findings-emnlp.296v3" hash="626fee76" date="2021-05-04">abstract correction</revision>
     </paper>
```

Comment:
This might be displayed some day, so let's write it as "Abstract correction." or "Updated abstract."